Example/Firebase.xcodeproj/xcshareddata/xcschemes/InstanceID_Tests_iOS.xcscheme

```diff
@@ -5,6 +5,19 @@
    <BuildAction
       parallelizeBuildables = "YES"
       buildImplicitDependencies = "YES">
+      <BuildActionEntries>
```

Comment:
This diff seems to be generated by AppCode and always shows up for me, even after reverting for a second.

Reply:
I also decided to leave the two random changes by `script/style.sh` in this PR. You are welcome :)